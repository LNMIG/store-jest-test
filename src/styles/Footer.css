.Footer {
  display: flex;
  justify-content: space-between;
  background-color: #33b1ff;
  margin-top: 1rem;
  padding-left: 0.5rem;
  padding-right: 0.5rem;
  border-radius: 0.5rem;
  font-weight: bold;
  letter-spacing: 0.15rem;
  color: #ffffff;
}
