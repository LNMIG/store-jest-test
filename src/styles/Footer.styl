.Footer
  display flex
  justify-content space-between
  color #33b1ff