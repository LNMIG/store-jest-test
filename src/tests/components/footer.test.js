import React from "react";
import { create } from "react-test-renderer"
import { render, screen } from '@testing-library/react'
import Footer from '../../components/footer.jsx'

describe(('Testing footer component'), () => {
    test('Check footers components text rendering 1', () => {
        render(<Footer/>)
      expect(screen.getByText('T-shirt Store')).toBeInTheDocument()
    });
    test('Check footers components text rendering 2', () => {
        render(<Footer/>)
        expect(screen.getByText('Rights reserved?')).toBeInTheDocument()
      })
})

describe(('Footer snapshot'), () => {
    test("Checking component footer's UI", () => {
      const footer = create(<Footer/>)
      expect(footer.toJSON).toMatchSnapshot()
    })
})