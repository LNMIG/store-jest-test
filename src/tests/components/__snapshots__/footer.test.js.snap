// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot Checking component footer's UI 1`] = `[Function]`;
